device-types/HPE/Aruba-EC-M-P-FIPS.yaml

```diff
@@ -1,24 +1,24 @@
 ---
 manufacturer: HPE
-model: Aruba ec-m-p-fips
+model: Aruba EC-M-P-FIPS
 slug: hpe-aruba-ec-m-p-fips
 part_number: '201634'
 u_height: 1
 is_full_depth: true
 airflow: front-to-rear
+weight: 22
+weight_unit: lb
 console-ports:
   - name: console
     type: de-9
     label: console
 power-ports:
```

Comment:
Hi Thank you for your contribution.
Are these modular PSUs? If so they should be modeled as `module-bays` with the required PSUs added to the modules folder if they don't already exist.